### PR TITLE
add regression testing for grid unit models

### PR DESCRIPTION
**Pull request by tskille**

### CODE REVIEW

**Comment by akva2** [comment]
jenkins build this update_data please

**Review by bska — COMMENTED** [review]
This mostly looks good to me, but I think there's some copy-paste leftover in the code to update the FIELD cases.

File: tests/update_reference_data.sh [hunk 1]
```diff
@@ -202,6 +202,12 @@ tests[krnum_02z]="flow krnum KRNUM-02Z"
 tests[krnum_03x]="flow krnum KRNUM-03X"
 tests[krnum_03y]="flow krnum KRNUM-03Y"
 tests[krnum_03z]="flow krnum KRNUM-03Z"
+tests[model_field_gridunit_cm]="flow gridunit M_FIELD_GRID_CM"
+tests[model_field_gridunit_cm]="flow gridunit M_FIELD_GRID_FEET"
+tests[model_field_gridunit_cm]="flow gridunit M_FIELD_GRID_METRES"
```
[the remark this answers is not in the document]
Comment (akva2) [reply tests/update_reference_data.sh:207]: indeed. i killed the started build.

[the remark this answers is not in the document]
Comment (tskille) [reply tests/update_reference_data.sh]: Thanks, should be fixed now.

**Comment by bska** [comment]
jenkins build this update_data please

**Pull request merged**